_data/publications/maricq-osdi18.yml

```diff
@@ -1,6 +1,6 @@
 title: "Taming performance variability"
 link: https://www.usenix.org/conference/osdi18/presentation/maricq
-date: Oct 8-10, 2018
```

Comment:
Not sure how a publication can have a range, and this was not understood by Date anyway.